### ISSUE TITLE
"Raw" judgehost image?

### ISSUE THREAD

**Issue by SolalPirelli** [DOMjudge/domjudge-packaging#15]
Opinionated request: please provide a "raw" judgehost image without anything that's not strictly needed for the judgehost, i.e. no Java, C++, etc. 

Right now it takes quite a while to create a judgehost, and it seems like part of it is installing software that won't be used or will be overwritten with different versions anyway. :-)

**Comment by nickygerritsen** [comment]
Do you mean as a Docker image? For example that we create `domjudge/judgehost:5.3.2-default` and `domjudge/judgehost:5.3.2-base` (or similar), where `-default` contains the programming languages and `-base` only the base needed? Note that for the latter we then can also *not* provide the chroot yet, as that needs the programming language binaries. This will make a big difference in image size, but will require you to always run it manually.

Which part of creating the judgehost takes long? It should be really fast except for downloading the image.

**Comment by SolalPirelli** [comment]
> Do you mean as a Docker image?

Yes. Sorry for the ambiguity. I'm fine with creating the chroot manually (maybe by running a script provided in the Docker image already).

> Which part of creating the judgehost takes long? It should be really fast except for downloading the image.

Exactly - the downloading part.

**Comment by nickygerritsen** [comment]
So we discussed this and think that this is not something we should or will provide ourselves currently. If you like this feature and can implement it yourself, feel free to open a PR and we'll have a look.
